testing/integration/ess/fleetserver_fips_test.go

```diff
@@ -1,106 +0,0 @@
-// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
-// or more contributor license agreements. Licensed under the Elastic License 2.0;
-// you may not use this file except in compliance with the Elastic License 2.0.
-
-//go:build integration
-
-package ess
-
-import (
-	"context"
-	"encoding/json"
-	"net/http"
-	"net/url"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/elastic/elastic-agent/pkg/testing/define"
-	"github.com/elastic/elastic-agent/pkg/testing/tools/fleettools"
-	"github.com/elastic/elastic-agent/testing/integration"
-)
-
-// TestFIPSAgentConnectingToFIPSFleetServerInECHFRH ensures that a FIPS-capable Elastic Agent
-// running in an ECH FRH (FedRamp High) environment is able to successfully connect to its
-// own local Fleet Server instance (which, by definition should also be FIPS-capable and
-// running in the ECH FRH environment).
-func TestFIPSAgentConnectingToFIPSFleetServerInECHFRH(t *testing.T) {
```

Comment:
Are we getting rid of this test entirely?

Reply:
Yes, because what it covers is being covered by the `ensureFleetServerInDeploymentIsHealthyAndFIPSCapable` function in the new unified test. That said, I need to update that function to directly check ES docs, so let me push up a commit for that.

Reply:
> That said, I need to update that function to directly check ES docs, so let me push up a commit for that.

20c214706b